web/concrete/single_pages/dashboard/blocks/stacks/view.php

```diff
@@ -31,10 +31,19 @@
         }
     }
 
+    $isGlobalArea = false;
+    if ($stack->getStackType() == Stack::ST_TYPE_GLOBAL_AREA) {
+        $isGlobalArea = true;
+    }
+
     ?>
 
     <div class="ccm-dashboard-header-buttons">
+        <?php if ($isGlobalArea) : ?>
+        <a href="<?=URL::to('/dashboard/blocks/stacks/view_global_areas')?>" data-dialog="add-stack" class="btn btn-default"><i class="fa fa-angle-double-left"></i> <?=t("Back to Global Areas")?></a>
+        <?php else: ?>
         <a href="<?=URL::to('/dashboard/blocks/stacks')?>" data-dialog="add-stack" class="btn btn-default"><i class="fa fa-angle-double-left"></i> <?=t("Back to Stacks")?></a>
+        <?php endif; ?>
```

Comment:
Please use brackets for your conditionals, alternate conditional syntax may be dropped in upcoming versions of PHP (my fingers are crossed).